ccd-definition/AuthorisationComplexType/caseDocuments/AdditionalApplicationsBundle.json

```diff
@@ -2040,5 +2040,1522 @@
         "CRUD": "CRUD"
       }
     ]
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "additionalApplicationsBundle",
+    "ListElementCode": "c2DocumentBundleConfidential",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[CAFCASSSOLICITOR]",
+          "caseworker-publiclaw-cafcass",
+          "caseworker-publiclaw-courtadmin",
+          "caseworker-publiclaw-systemupdate",
+          "[BARRISTER]"
```

Comment:
are you sure [CAFCASSSOLICITOR] and [BARRISTER] can access confidential c2 document?

Reply:
Maybe you are right if following confidential rules. Removed their access